test/src/unit-vfs.cc

```diff
@@ -547,6 +547,26 @@ TEMPLATE_LIST_TEST_CASE("VFS: File I/O", "[vfs][uri][file_io]", AllBackends) {
   }
 }
 
+TEST_CASE("VFS: Test end-to-end", "[.][vfs][e2e]") {
```

Comment:
This test will run with `./tiledb_unit [vfs]`, is that the intention here? If not we could use `[.][vfs-e2e]` or `[.vfs-e2e]` to avoid the test being selected with the more general `[vfs]` tag.

Reply:
Nice catch. Fixed.